drivers/hmis/app/models/hmis/hud/household.rb

```diff
@@ -17,14 +17,21 @@ class Hmis::Hud::Household < Hmis::Hud::Base
   alias_attribute :household_id, :HouseholdID
 
   replace_scope :viewable_by, ->(user) do
-    viewable_households = joins(:enrollments).
-      merge(Hmis::Hud::Enrollment.viewable_by(user)). # does Data Source filter
-      pluck(:HouseholdID)
-
-    where(HouseholdID: viewable_households)
+    # correlated subquery
+    p_t = Hmis::Hud::Project.arel_table
+    hh_t = Hmis::Hud::Household.arel_table
```

Comment:
These should both already be available given the inclusion of `::Hmis::Concerns::HmisArelHelper` above.